src/Services/Podcasts/Podcast.MinimalAPI/appsettings.Development.json

```diff
@@ -7,7 +7,7 @@
     }
   },
   "ConnectionStrings": {
-    "PodcastDb": "",
+    "PodcastDb": "Server=localhost, 5433;Database=Podcast;User Id=sa;Password=Pass@word",
```

Comment:
Isn't this going to get flagged?

Reply:
nope, it is already in my other one, it is just for the localhost docker